src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java

```diff
@@ -290,7 +289,8 @@ public int getProcessQueueMaxLength()
 	@Override
 	public float getMinProcessDistance(MultiblockProcess<MultiblockRecipe> process)
 	{
-		return .5f;
+		float maxTicks = BottlingProcess.getMaxProcessTick();
+		return 1f-(BottlingMachineRenderer.getTransportTime(maxTicks)+BottlingMachineRenderer.getLiftTime(maxTicks))/maxTicks;
```

Comment:
I would move these functions to the TE. I suspect that the current implementation crashes on dedicated servers, and even if it doesn't I would avoid referencing the tile render class in server-side code.

Reply:
good point, done